docs/extend/modular-resources/examples/custom-arm.md

```diff
@@ -0,0 +1,252 @@
+
+See [Arm Configuration](/components/arm/#configuration) for the current list of built-in models the RDK provides.
+
+{{% /alert %}}
+
+If you have a robot arm that is not already supported by the RDK, creating a custom module for it allows you to [program](/program/) and control it with the [arm API](/components/arm/#api), or use it with Viam [services](/services/) such as [Motion Planning](/services/motion/), just as you would with a built-in model.
+
+Follow these instructions to implement a model of [arm component](/components/arm/) that is not built into the RDK:
+
+- [Get your arm's kinematics file](#get-your-arms-kinematics-file)
+- [Create a custom arm model as a modular resource](#create-a-custom-arm-model-as-a-modular-resource)
+  - [Code a new resource model](#code-a-new-resource-model)
+  - [Code a main entry point program](#code-a-main-entry-point-program)
+  - [Compile the module into an executable](#compile-the-module-into-an-executable)
+- [Configure the module and modular resource on your robot](#configure-the-module-and-modular-resource-on-your-robot)
+
+This guide uses Viam's Python SDK to code the arm module, but if you want to use the Go Client SDK, you can.
+Follow [this guide](/extend/modular-resources/create/#code-a-new-resource-model) and select **Go** to learn how to code a module providing a new arm model in Go.
+
+## Get your arm's kinematics file
+
+The way arms move across space is more complicated than Viam's other [components](/components/).
+Because of this, an arm, unlike other components, requires a [kinematic configuration file](/internals/kinematic-chain-config/) describing its geometry.
+This provides necessary information for the built-in [Motion Service](/services/motion/) to work with the arm, on top of the client-inaccessible and mostly static [Frame System Service](/services/frame-system/).
+
+**Find a pre-built kinematics file:**
+
+- `viam-server` will work with <file>URDF</file> (United Robot Description Format) kinematics files, which are currently the standard for ROS drivers.
+Many <file>URDF</file> "robot descriptions" for industrial robot arm models can be found on [GitHub](https://github.com/) that could work with the arm on Viam.
+
+**Create your own kinematics file:**
+
+- Follow the instructions on [Configure Complex Kinematic Chains](/internals/kinematic-chain-config/) to write a file detailing the geometry of your arm.
+  - Use the [Spatial Vector Algebra (SVA)](/internals/kinematic-chain-config/#kinematic-parameters) kinematic parameter type.
+  - Define the parameters in a </file>.json</file> file.
+  - Follow the Frame System's guide to [Configure a Reference Frame](/services/frame-system/frame-config/) when working out the relative [orientation](/internals/orientation-vector/) of the `"links"` on your arm.
```

Comment:
```suggestion
  - Follow the Frame System's guide to [Configure a Reference Frame](/services/frame-system/frame-config/) when working out the relative [orientations](/internals/orientation-vector/) of the `links` on your arm.
```
[opt] "links" could be completely unstyled because they are just...things? concepts? Not sure how to put it. I mean they're not just code entities. But per your comment on my original suggestion here, I think if you want them styled in some way to call them out, they don't need both quotes and backticks.
[nit] Orientations is plural here I believe